sarif/PhylumExt.toml

```
@@ -0,0 +1,7 @@
+name = "sarif"
+
+[permissions]
+write = false 
+read = false 
+run = false 
+net = true
```

False is the default. No need to clutter the file

```suggestion
net = true
```